tensorflow/compiler/xla/debug_options_flags.cc

```diff
@@ -72,7 +72,11 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
 
   opts.set_xla_allow_excess_precision(true);
   opts.set_xla_force_host_platform_device_count(1);
+#if !TENSORFLOW_USE_ROCM
```

Comment:
nit: we always use `#if TENSORFLOW_USE_ROCM`

Reply:
fixed